README.md

```diff
@@ -1,206 +1,60 @@
-# Notary Project Overview
-
-A current overview of Notary v2, including a collection of requirements & scenarios, framing the scope of the Notary v2 project.
+# The Notary Project Overview
+
+The Notary Project aims to provide enterprise-grade solutions and cross-industry standards for securing software supply chain. The following are current repositories, aka sub-projects, under the Notary Project umbrella in an alphabetical order:
+
+| Repository                                                               | Description                                                                                                                                             |
+| ------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [.github](https://github.com/notaryproject/.github)                      | The repository that contains the Notary Project governance documents                                                                                    |
+| [meeting-notes](https://github.com/notaryproject/meeting-notes)          | The repository that contains archived meeting notes                                                                                                     |
+| [notary](https://github.com/notaryproject/notatry)                       | The repository that is an implementation of TUF that runs next to a container registry and adds the ability to sign and verify content in the registry  |
+| [notaryproject](https://github.com/notaryproject/notatryproject)         | The repository that contains the Notary Project requirements and specifications                                                                         |
+| [notaryproject.dev](https://github.com/notaryproject/notatryproject.dev) | The repository that contains documents and styles for the Notary Project website                                                                        | 
+| [notation](https://github.com/notaryproject/notation)                    | The repository that is an implementation of the Notary Project specification and provides a CLI tooling to sign and verify artifacts                                                                                 |
+| [notation-go](https://github.com/notaryproject/notation-go)              | The repository that is an implementation of the Notary Project specification and provides the library supports of signing and verifying artifacts                    |
+| [notation-core-go](https://github.com/notaryproject/notation-core-go)    | The repository that is an implementation of the Notary Project signature specification and provides the library supports                                                                             |
```

Comment:
This is likely an area for improved clarity, as these all reference Notation, but only the third repository defines this as the Notary Project Signature Specification. The other two do not despite stating that the first (notation) performs signing and verification and the second (notation-go) is limited to support of said signing and verification.  So is the implication here that the first two of these three do not implement the signature specification only the third?

Reply:
I provided additional comments on https://github.com/notaryproject/.github/pull/32 that mirror this.

Reply:
overhauled readme.md per https://github.com/notaryproject/.github/issues/35